tests/compatibility-tests/src/test/java/org/apache/activemq/artemis/tests/compatibility/PagingCounterTest.java

```diff
@@ -51,8 +51,12 @@ public static Collection getParameters() {
       //      combinations.add(new Object[]{SNAPSHOT, ONE_FIVE, ONE_FIVE});
       //      combinations.add(new Object[]{ONE_FIVE, ONE_FIVE, ONE_FIVE});
 
-      combinations.add(new Object[]{null, TWO_TWENTYTWO_ZERO, SNAPSHOT});
-      combinations.add(new Object[]{null, SNAPSHOT, TWO_TWENTYTWO_ZERO});
+      if (getJavaVersion() <= 22) {
+         // 2.22.0 servers fail on JDK23+ without workarounds.
```

Comment:
I have seen you keeping Twenty two on other tests.

Isn't there any server we can keep in its replacement. I would like to just keep verifying the compatibility between SNAPSHOT and *any* previous version.